server/sqlstore/playbook_run_test.go

```diff
@@ -1205,6 +1170,40 @@ func TestGetParticipantsActiveTotal(t *testing.T) {
 	}
 }
 
+func TestUpdateFollowers(t *testing.T) {
+	followersOld := []string{"1", "2", "3"}
+	followersNew := []string{"1", "5", "4"}
+
+	teamID := model.NewId()
+
+	for _, driverName := range driverNames {
+		db := setupTestDB(t, driverName)
+		playbookRunStore := setupPlaybookRunStore(t, db)
+		setupChannelsTable(t, db)
+		setupTeamMembersTable(t, db)
+
+		run := NewBuilder(t).
+			WithCurrentStatus(app.StatusInProgress).
+			WithTeamID(teamID).
+			ToPlaybookRun()
+
+		returned, err := playbookRunStore.CreatePlaybookRun(run)
+		require.NoError(t, err)
+		// add run followers
+		for _, f := range followersOld {
+			err = playbookRunStore.Follow(returned.ID, f)
+			require.NoError(t, err)
+		}
+
+		err = playbookRunStore.UpdateFollowers(returned.ID, followersNew)
+		require.NoError(t, err)
+		followers, err := playbookRunStore.GetFollowers(returned.ID)
+		require.NoError(t, err)
+		sort.Strings(followers)
+		require.Equal(t, []string{"1", "4", "5"}, followers)
```

Comment:
Should we use here `followersNew` (probably with a prior sorting, too)?
```suggestion
		require.Equal(t, followersNew, followers)
```